src/domains/load-test/01_grafana.tf

```diff
@@ -0,0 +1,21 @@
+module "grafana_managed" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//grafana?ref=feature/grafana"
+
+  name = format("%s-grafana", var.env_short)
+
+  resource_group_name = var.terraform_remote_state_core.resource_group_name
```

Comment:
Please put the grafana in its own resource group, and not use the one used by terraform state

Reply:
I created a new one. Is it right?

Reply:
yes